src/helpers/utilities.ts

```diff
@@ -7,8 +7,11 @@ type BigNumberish = number | string | BigNumber;
 interface Dictionary<T> {
   [index: string]: T;
 }
+// <<<<<<< HEAD
+// =======
```

Comment:
👀 👀 👀 

Gotta be more careful with merge conflicts

Reply:
yeah, the frequent updating of the same ones PRs hypnotized me